google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -3321,6 +3532,30 @@ private boolean canCreateSession() {
     }
   }
 
+  private void maybeCreateMultiplexedSession() {
+    synchronized (lock) {
+      if (!numMultiplexedSessionsBeingCreated) {
+        logger.log(Level.FINE, String.format("Creating multiplexed sessions"));
+        try {
+          numMultiplexedSessionsBeingCreated = true;
+          sessionClient.createMultiplexedSession(multiplexedSessionConsumer);
+        } catch (Throwable ignore) {
+          // such an exception will never be thrown. the exception will be passed onto the consumer.
+        }
+      }
+    }
+  }
+
+  private void handleMultiplexedSessionsFailure(SpannerException e) {
```

Comment:
nit: rename to `handleCreateMultiplexedSessionFailure()`

Reply:
Done.